src/FirebirdSql.EntityFrameworkCore.Firebird.FunctionalTests/Query/NorthwindSetOperationsQueryFbTest.cs

```diff
@@ -37,6 +37,14 @@ public override Task Union_Select_scalar(bool async)
 		return base.Union_Select_scalar(async);
 	}
 
+	// Not supported on FB 3 and 4
```

Comment:
There's `FbTestStore.ServerLessThan4`. So you can create similar for 5 and use it here to run the test on FB 5 only.